cluster-agent/go.sum

```diff
@@ -475,6 +475,8 @@ github.com/itchyny/timefmt-go v0.1.2/go.mod h1:0osSSCQSASBJMsIZnhAaF1C2fCBTJZXrn
 github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99 h1:BQSFePA1RWJOlocH6Fxy8MmwDt+yVQYULKfN0RoTN8A=
 github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99/go.mod h1:1lJo3i6rXxKeerYnT8Nvf0QmHCRC1n8sfWVwXF2Frvo=
 github.com/jessevdk/go-flags v1.4.0/go.mod h1:4FA24M0QyGHXBuZZK/XkWh8h0e1EYbRYJSGM75WSRxI=
+github.com/jgwest/managed-gitops/cluster-agent v0.0.0-20211027023712-42af25b249bf h1:vj07onsiXmPR2Tsd0IktyAJTG/bOgNOwgKW1ObkAMfY=
```

Comment:
Likewise guessing these will be removed when the `jgwest` referenced is removed from go.mod.